XIVSlothCombo/Combos/CustomComboPreset.cs

```diff
@@ -1547,6 +1547,10 @@ public enum CustomComboPreset
         [CustomComboInfo("Burst Strike to Bloodfest Feature", "Replace Burst Strike with Bloodfest if you have no powder gauge.", GNB.JobID)]
         GNB_BS_Bloodfest = 7402,
 
+        [ParentCombo(GNB_BS)]
+        [CustomComboInfo("Reign combo on Burst Strike Feature", "Adds Reign combo to Burst Strike when under No Mercy and when Double Down & Gnashing Fang are on CD.", GNB.JobID)]
+        GNB_BS_Reign = 7403,
```

Comment:
Same ID as line 1556.

Reply:
Mistake on my end. Sorry about that. Was a last minute add in, but no excuses.